code/__DEFINES/traits/declarations.dm

```diff
@@ -309,4 +308,28 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /// For unit testing, all do_afters set on this mob complete instantly and do not sleep
 #define INSTANT_DO_AFTER "instant_do_after"
 
+
+// Radiation defines
+
+/// Marks that this object is irradiated
+#define TRAIT_IRRADIATED "iraddiated"
+
+/// Immune to being irradiated
+#define TRAIT_RADIMMUNE "rad_immunity"
+
+/// Harmful radiation effects, the toxin damage and the burns, will not occur while this trait is active
+#define TRAIT_HALT_RADIATION_EFFECTS "halt_radiation_effects"
+
+//If the species gains good things from radiation. See: Dionae
+#define TRAIT_RADBOOSTER "rad_booster"
```

Comment:
The name of this trait doesn't really make much sense. I also think that theres a bit of an issue where if you have a species that heals from rads and forget to put the trait that stops you from getting the irradiated component then your species will get permanent immunity which is a bit of a trap for future coders but I guess its fine

Reply:
I see what the intention is for this now, TRAIT_ALWAYS_PROCESS_RADS, TRAIT_BENEFICIAL_RADS or something might make more sense